pvops/text/utils.py

```diff
@@ -80,3 +80,25 @@ def remap_attributes(om_df, remapping_df, remapping_col_dict,
               "{sum(df[ATTRIBUTE_COL].isna())}")
 
     return df
+
+def replace_word_with_pv_term(word, pv_reference_dict):
```

Comment:
Unclear what the difference from `text_replace_key_terms` is. Consider merging the two functions here.

Reply:
renamed `text_replace_key_terms()` to `remap_words_in_text()` for clarity